api/admin.go

```diff
@@ -325,7 +332,12 @@ func (a *API) adminUserDelete(w http.ResponseWriter, r *http.Request) error {
 	instanceID := getInstanceID(ctx)
 	adminUser := getAdminUser(ctx)
 
-	err := a.db.Transaction(func(tx *storage.Connection) error {
+	var err error
+	params := &adminUserDeleteParams{}
+	q := r.URL.Query()
+	params.IsSoftDelete = q.Get("is_soft_delete")
```

Comment:
Wondering if we should default to soft deletes instead and have something like `force_delete`? Seems like it might better guard against accidents. Atlassian's major outage and their [post outage resolution](https://www.atlassian.com/engineering/post-incident-review-april-2022-outage)  to default to universal soft deletes came to mind. 

But defer to infra/users of the endpoint.